include/atomics/subnet_cadmium.hpp

```diff
@@ -1,7 +1,18 @@
-	/**
-* Cristina Ruiz Martin
-* ARSLab - Carleton University
+/** Header file for subnet
```

Comment:
wrong commenting standard. should be like this

"/**  \brief This main file implements the operation of subnet file"